src/pages/index.js

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
-import { Link } from 'gatsby';
+import { graphql } from 'gatsby';
 
 import Layout from '../components/layout';
 
-const IndexPage = () => (
+const IndexPage = ({ data }) => (
   <Layout>
-    <h1>Hi people</h1>
-    <p>Welcome to your new Gatsby site.</p>
-    <p>Now go build something great.</p>
-    <Link to="/page-2/">Go to page 2</Link>
+    <h2>{data.ploneDocument.title}</h2>
```

Comment:
Semantically this should be H1.

If there is H1 in layout, maybe it should be semantically something else?

Reply:
H1 was used in the header, hence I decided to go with h2. What would be the alternative here?

Reply:
If every page has the same H1, it becomes meaningless.

I believe, nowadays it is ok to have multiple H1 on the same page, if they are wrapped with `<article/>` or similar semantic container.

Simpler alternative is to use semantically meaningless tag (div or span) in header "H1" and only make it look big with CSS.